include/zephyr/net/coap_link_format.h

```diff
@@ -34,6 +34,12 @@ int coap_well_known_core_get(struct coap_resource *resource,
 			     struct coap_packet *response,
 			     uint8_t *data, uint16_t len);
 
+int coap_well_known_core_get_len(struct coap_resource *resources,
```

Comment:
missing docstring (or maybe it's not meant to be a public API? docstring also missing for coap_well_known_core_get() just above.